python/tests/mlir/ast_list_int.py

```diff
@@ -17,7 +17,7 @@ def test_list_int():
     def oracle(register: cudaq.qview, auxillary_qubit: cudaq.qubit,
                hidden_bitstring: list[int]):
         # Also test out len() here, should convert to stdvec_size
-        x = len(hidden_bitstring)
+        n = len(hidden_bitstring)
```

Comment:
It seems that before, x.ctrol was assuming x to be the global x gate despite a variable shadowing it. This does not match python behavior, and we either need to forbit the creation of any identifiers that would shadow built in gates, or we respect python behavior (at least more and more as we improve the bridge)